app/components/battleship.hbs

```diff
@@ -0,0 +1,46 @@
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
```

Comment:
Feel like this component could use a different name. Since we are likely to have an actual... battleship...
TitleHeader, NavHeader... whatever works

Reply:
Ah, sure. Will call it `ship-svg`.